tests/test-distribution/test-distribution-common/src/test/java/org/eclipse/jetty/tests/distribution/DistributionTests.java

```diff
@@ -200,26 +191,27 @@ public void testSimpleWebAppWithJSPandJSTL(String env) throws Exception
         }
     }
 
-    @Disabled //TODO runtime can't find org.objectweb.asm on module path?
     @ParameterizedTest
-    @ValueSource(strings = {"ee9", "ee10"})
+    @ValueSource(strings = {"ee10"})
     public void testSimpleWebAppWithJSPOnModulePath(String env) throws Exception
     {
+        // Testing with env=ee9 is not possible because jakarta.transaction:1.x
+        // does not have a proper module-info.java, so JPMS resolution will fail.
```

Comment:
Sounds like a new issue needs to be filed for this

Reply:
A new issue to what project?

I don't think it can be fixed, as `jakarta.transaction:1.x` should remain Java 8 compatible (so no `module-info.java`) as per JEE 9.
It only has an `Automatic-Module-Name` but that is not enough to run with full JPMS.

I don't think the JakartaEE project will modify the jar, since they have already fixed it in `jakarta.transaction:2.x` (and in fact, our ee10 tests pass without problems).